runtime/tr.source/trj9/build/rules/aix-xlc/filetypes.mk

```diff
@@ -103,34 +103,19 @@ endef # DEF_RULE.ipp
 
 RULE.ipp=$(eval $(DEF_RULE.ipp))
 
-# Compile .aspp file into .ipp file
-define DEF_RULE.aspp
-$(1).ipp: $(2) | jit_createdirs
-	$$(ASPP_CMD) $$(ASPP_FLAGS) $$(patsubst %,-D%,$$(ASPP_DEFINES)) $$(patsubst %,-I'%',$$(ASPP_INCLUDES)) -E $$< > $$@
-
-JIT_DIR_LIST+=$(dir $(1))
-
-jit_cleanobjs::
-	rm -f $(1).ipp
-
-$(call RULE.ipp,$(1),$(1).ipp)
-endef
-
-RULE.aspp=$(eval $(DEF_RULE.aspp))
-
 #
 # Compile .spp file into .o file
 #
 define DEF_RULE.spp
-$(1).aspp: $(2) | jit_createdirs
-	$$(SPP_CMD) $$(SPP_FLAGS) $$< $$@
+$(1).ipp: $(2) | jit_createdirs
+	$$(ASPP_CMD) $$(ASPP_FLAGS) $$(patsubst %,-D%,$$(ASPP_DEFINES)) $$(patsubst %,-I'%',$$(ASPP_INCLUDES)) -E $$< > $$@
 
 JIT_DIR_LIST+=$(dir $(1))
 
```

Comment:
there will be no *.aspp (intermediate or temporary) files after this work.  we should not retain a step called ASPP.  i.e. your ASPP_CMD should be called SPP_CMD, since *.spp files are the starting point.  Please look at gnu-xlc.mk for the clues.